### PR TITLE
Hakob/login Added: ReactRouter, PageRouter, SignIn/SignUp Pages + BE Implementation

### PR DESCRIPTION
**Pull request by Hakob3215**
Added React Router to FE + SignIn/SignUp implementation
 - I made the main display the page router
 - The game page is one of the routes "/game", which still uses App.js
 - The backend has an account router, which handles user account requests
 - Making an account has been implemented in FE/BE
   - However, email verification has not been implemented
   - Password hashing/encrypting HAS been implemented (bcrypt)
 - new .env variables for frontend (will send in discord)

### CODE REVIEW

File: frontend/src/PageRouter.js
Comment (jaden-yejun-lee) [inline frontend/src/PageRouter.js:1]: Since 'App' is now the 'game' page, could we rename the app file as 'game' to make it more clear?

**Review by jaden-yejun-lee — CHANGES_REQUESTED** [review]
app -> game

File: frontend/src/PageRouter.js
Comment (AnoldGH) [reply frontend/src/PageRouter.js:1]: I second this

**Review by AnoldGH — APPROVED** [review]
LGTM

**Review by jaden-yejun-lee — APPROVED** [review]
Looks good!

**Pull request merged**